lib/Basic/LangOptions.cpp

```diff
@@ -115,6 +117,12 @@ std::pair<bool, bool> LangOptions::setTarget(llvm::Triple triple) {
   case llvm::Triple::ArchType::aarch64:
     addTargetConfigOption("arch", "arm64");
     break;
+  case llvm::Triple::ArchType::ppc64:
+    addTargetConfigOption("arch", "powerpc64");
+    break;
+  case llvm::Triple::ArchType::ppc64le:
+    addTargetConfigOption("arch", "powerpc64le");
+    break;
```

Comment:
This part needs a test like `test/BuildConfigurations/x64LinuxTarget.swift` for each supported triple.